lib/src/locale/locale.dart

```diff
@@ -103,8 +104,13 @@ const GermanDurationLocale germanLocale = GermanDurationLocale();
 /// [DurationLocale] for Dutch language
 const DutchDurationLocale dutchLocale = DutchDurationLocale();
 
-/// [DurationLocale] for Chinese language
-const ChineseDurationLocale chineseLocale = ChineseDurationLocale();
+/// [DurationLocale] for Chinese (Simplified) language
+const ChineseSimplifiedDurationLocale chineseSimplifiedLocale =
```

Comment:
This would mean a breaking change. Is it possible to keep the old name here? Or would that be wrong?